<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDACheckEnvDialog</class>
 <widget class="QDialog" name="QDACheckEnvDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="checkEnvBtn">
       <property name="text">
        <string>检查环境 (Check Env)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tryFixBtn">
       <property name="text">
        <string>尝试修复 (Try Fix)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTreeWidget" name="envTreeWidget">
     <column>
      <property name="text">
       <string>软件(Soft)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>环境变量(EnvVar)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>路径(Path)</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
